documentation/3.4/content/release-notes.md

```diff
@@ -55,6 +56,13 @@ draft: false
 
 ### Change log
 
+#### Operator 3.4.4
+
+* Support added to specify resource requests and limits for Monitoring Exporter sidecar containers.
+* Resolved an issue that prevented the generation of the domain rolling completed event.
```

Comment:
the domain rolling completed event -> again, having a hard time understanding this language -> So, there was an issue that prevented the generation of an event called domain rolling completed? If so, it might help to put `domain rolling completed` in code font.

Reply:
Done